lib/_http_outgoing.js

```diff
@@ -183,11 +190,9 @@ OutgoingMessage.prototype._storeHeader = function(firstLine, headers) {
     messageHeader: firstLine
   };
 
-  var field, value;
-
   if (headers) {
     var keys = Object.keys(headers);
-    var isArray = util.isArray(headers);
+    var isArray = Array.isArray(headers);
```

Comment:
We could probably get a further speedup from switching these to `{var} instanceof Array` - http://jsperf.com/array-isarray-vs-instanceof-array/5

benchmark puts it at 2x faster